options.go

```diff
@@ -117,5 +148,7 @@ func mergeOptions(opts VerifyOptions, options ...VerifyOption) VerifyOptions {
 		opts.rootVerifier = identity.NewDisabledVerifier()
 	}
 
+	opts.allIdentitiesParamsSet = opts.maxIdentitiesCount == -1 || !opts.lastIdentityCreationTimestamp.IsZero()
```

Comment:
```suggestion
	opts.allIdentitiesParamsSet = opts.maxIdentitiesCount != -1 || !opts.lastIdentityCreationTimestamp.IsZero()
```

Reply:
No, it works as `count.isZero() OR timestamp.isZero()`. -1 is default valud for counter, because validation.IsEmpty works incorrect if the counter field is 0